lib/config.js

```diff
@@ -154,6 +154,8 @@ function getLocalConfig(thisConfig, directory) {
 
         if (personalConfig) {
             return ConfigOps.merge(config, personalConfig);
+        } else if (thisConfig.options.rules && thisConfig.options.rules !== {}) {
```

Comment:
Sorry, I should have been more careful.  I'm rushing and that's never a good thing.